portal-kernel/src/com/liferay/portal/kernel/upload/UploadServletRequestSettingsUtil.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.kernel.upload;
+
+import com.liferay.portal.kernel.util.ServiceProxyFactory;
+
+/**
+ * @author Pei-Jung Lan
+ */
+public class UploadServletRequestSettingsUtil {
```

Comment:
That's so weird, I always do it with git rebase and it works fine. Maybe it's a temporary issue with GitHub?

Let me know if you find out what it was.